cmd/root.go

```diff
@@ -0,0 +1,34 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	flagQuiet = "quiet"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "go-ci-fuzz",
+	Short: "Run Go Fuzz targets in CI systems",
+	Long: `go-ci-fuzz implements missing functionality in 'go test -fuzz' such as
+- running multiple test targets in a single command
+- extracting failed outputs
+- corpus management
+`,
```

Comment:
For the `Long` description we do not need to hard code the name of the binary command (`go-ci-fuzz`), as it becomes redundant when running the command. 

```shell
~/code ❯ go-ci-fuzz
go-ci-fuzz implements missing functionality in 'go test -fuzz' such as
- running multiple test targets in a single command
- extracting failed outputs
- corpus management

Usage:
  go-ci-fuzz [command]
...
```

So a more standard long description could be:
```go
Long: `Implements missing functionalities in 'go test -fuzz' such as
- run multiple test targets in a single command
- extract failed outputs
- corpus management
```
(also `functionalities` instead of `functionality`; `extract` and `run` instead of `*ing` equivalent)